src/Illuminate/Contracts/Session/Session.php

```diff
@@ -47,6 +47,14 @@ public function save();
      */
     public function all();
 
+    /**
+     * Get a subset of the session data.
+     *
+     * @param array $keys
+     * @return array
+     */
+    public function only($keys);
```

Comment:
Is this method on the contract really necessary? The contract is still fulfilled without this method. The method seems like something that could exist on the internal framework class but not necessary to provide the basic needs of a `Session` object.

Reply:
In that regard, if this method on the contract was removed, it could be targeted to the 5.8 branch.

Reply:
Sure, seems good to me. Rebased and targeted 5.8